components/decimal/benches/fixed_decimal_format.rs

```diff
@@ -34,7 +33,7 @@ fn overview_bench(c: &mut Criterion) {
             // ranging from -1e9 to 1e9.
             let fdf = FixedDecimalFormatter::try_new_unstable(
                 &provider,
-                &Locale::UND.into(),
+                &Default::default(),
```

Comment:
Thought, here and elsewhere: `Locale::UND.into()` is more clear than `Default::default()`; I think `&Locale::default().into()` would be the most clear. I also don't know whether `Preferences` will impl Default. @zbraniecki might have thoughts on this.

Reply:
+1

Reply:
We use `Default::default()` in constructors basically everywhere in tests. I don't see any value in changing a handful of invocations to `Locale::default().into()` now, only to change it again when we have preferences. If preferences don't implement `Default`, we'll have to change these call sites then anyway, so this won't be forgotten, if they do, then `Default::default()` is perfectly fine.